idb/postgres/postgres.go

```diff
@@ -850,15 +852,27 @@ func (db *IndexerDb) Transactions(ctx context.Context, tf idb.TransactionFilter)
 
 	round, err := db.getMaxRoundAccounted(ctx, tx)
 	if err != nil {
-		tx.Rollback(ctx)
+		// Because we return a channel into a "callWithTimeout" function,
+		// We need to make sure that rollback is called before close()
+		// otherwise we can end up with a situation where "callWithTimeout"
+		// will cancel our context, resulting in connection pool churn
+		if rerr := tx.Rollback(ctx); rerr != nil {
+			db.log.Printf("rollback error: %s", rerr)
```

Comment:
should this use `%v` or `%w`?

Reply:
Since it isn't returning/wrapping the error, I think it is fine here to convert to string.

Reply:
Is there some documentation or standard that recommends %v over %s in this scenario?

Reply:
Not sure, probably `%s` is fine.